src/fmeval/eval_algorithms/qa_accuracy.py

```diff
@@ -272,8 +405,21 @@ def __init__(self, eval_algorithm_config: QAAccuracyConfig = QAAccuracyConfig())
         :param eval_algorithm_config: QA Accuracy evaluation algorithm config.
         """
         super().__init__(eval_algorithm_config)
+
+        self.bertscore_model = BertscoreHelperModel(eval_algorithm_config.model_type_for_bertscore)
+        qa_accuracy_score = QAAccuracyScores(target_output_delimiter=eval_algorithm_config.target_output_delimiter)
+        bert_score = BertScoreWithDelimiter(
+            target_output_keys=[DatasetColumns.TARGET_OUTPUT.value.name],
+            model_output_keys=[DatasetColumns.MODEL_OUTPUT.value.name],
+            output_keys=[BERT_SCORE],
+            allow_duplicate_input_keys=True,
+            bertscore_model=self.bertscore_model,
+            target_output_delimiter=eval_algorithm_config.target_output_delimiter,
+        )
         self._eval_algorithm_config = eval_algorithm_config
-        self.transform = QAAccuracyScores(target_output_delimiter=eval_algorithm_config.target_output_delimiter)
```

Comment:
Removing `self.transform` is a breaking change. In hindsight, I should've made this private. Users really shouldn't be exposed to the internals of the transforms or pipelines when interacting with an eval algo. We should consider updating all of the eval algos such that these kinds of fields are private; this would require a major version bump before we release the changes.

Reply:
re-added self.transform